pkgs/desktops/gnome-3/misc/gpaste/fix-paths.patch

```diff
@@ -44,7 +44,7 @@
  g_paste_settings_init (GPasteSettings *self)
  {
      GPasteSettingsPrivate *priv = g_paste_settings_get_instance_private (self);
--    GSettings *settings = priv->settings = g_settings_new (G_PASTE_SETTINGS_NAME);
+-    GSettings *settings = priv->settings = create_g_settings ();
```

Comment:
Perhaps we should patch it here so people can use the file storage if they want:

https://github.com/Keruspe/GPaste/blob/53273955a754e17abb779e00e07d991cacf53c4a/src/libgpaste/settings/gpaste-settings.c#L1016